scripts/Iris.json

```diff
@@ -0,0 +1,88 @@
+{
+    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.
+
+",
```

Comment:
Interesting. I just confirmed that it doesn't show up on my machine if just the current JSON script is there. Ubuntu 16.04, Python 3.5.2. This is true both running from inside the repo and for outside the repo after running `retriever update` to get the current script that's now in `master`.